src/Model/Item.php

```diff
@@ -899,8 +899,9 @@ public static function insert($item, $notify = false, $dontcache = false)
 
 			// If its a post that originated here then tag the thread as "mention"
 			if ($item['origin'] && $item['uid']) {
+				DBA::update('post-thread-user', ['mention' => true], ['uri-id' => $item['parent-uri-id'], 'uid' => $item['uid']]);
 				DBA::update('thread', ['mention' => true], ['iid' => $parent_id]);
```

Comment:
Shouldn't this call be removed?

Reply:
This will be done in the next PR. Then all "thread" related code will be removed.